Runtime/Behaviours/CinemachineBrain.cs

```diff
@@ -679,9 +679,10 @@ private void UpdateFrame0(float deltaTime)
                         {
                             // Special case: if backing out of a blend-in-progress
                             // with the same blend in reverse, adjust the blend time
-                            if (frame.blend.CamA == activeCamera
```

Comment:
The bug is caused by this check here. When we are mid-blend, then frame.blend.CamA is not equal to activeCamera, because activeCamera is going to be a vcam, whereas frame.blend.CamA is going to be a Cinemachine.BlendSourceVirtualCamera (e.g. [Target CM Vcam] 95% from [Source CM Vcam]).